tech-specs/_gallery/src/index.css

```
@@ -0,0 +1,153 @@
+@import "tailwindcss";
+
+@theme {
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)